pkg/install/1-installresources.go

```diff
@@ -668,9 +664,29 @@ func (i *Installer) installResources(ctx context.Context) error {
 			return err
 		}
 	}
-
 	{
-		ipAddress := lbIP.String()
+		var ipAddress string
+		{
+			// Discover what private IP address was assigned to internal LB
+			lbName := "aro-internal-lb"
+			r, err := azureextended.ParseResourceGroupID(i.doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID)
+			if err != nil {
+				return err
+			}
+			lb, err := i.lbClient.Get(ctx, r.ResourceGroupName, lbName, "")
+			if err != nil {
+				return err
+			}
+			f := *lb.FrontendIPConfigurations
+			if f == nil || len(f) == 0 {
+				return fmt.Errorf("FrontendIPConfigurations in load balancer \"%s\" is nil or an empty slice", lbName)
+			}
+			privateIP := f[0].PrivateIPAddress
+			if privateIP == nil || *privateIP == "" {
+				return fmt.Errorf("PrivateIPAddress pointer in load balancer \"%s\" is nil or points to an empty string", lbName)
```

Comment:
nit: I think we can drop `\"` and just leave `%s` as we do this all around in the code.